sample-xml/app/src/test/java/co/nimblehq/sample/xml/ui/screens/xml/HomeFragmentTest.kt

```diff
@@ -28,9 +28,9 @@ class HomeFragmentTest : BaseFragmentTest<HomeFragment, FragmentHomeBinding>() {
     }
 
     @Test
-    fun `When initializing fragment, it displays the title correctly`() {
+    fun `When initializing fragment, it displays the recycler-view`() {
```

Comment:
The hyphen feels weird to me, perhaps `recycler view` instead? What do you think?
The rest LGTM 👌🏻